module/zfs/spa_misc.c

```diff
@@ -59,7 +59,9 @@
 #include <sys/kstat.h>
 #include "zfs_prop.h"
 #include <sys/zfeature.h>
-#include "qat.h"
+#ifdef __linux__
```

Comment:
Can we drop the `__linux__` wrapper here and make sure proper compatibility code is provided for the other platforms.